ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -0,0 +1,412 @@
+                                id="updateCredentials"
+                                isChecked={values.updateCredentials}
+                                onChange={(event, value) => onUpdateCredentialsChange(value, event)}
+                                onBlur={handleBlur}
+                                isDisabled={!isEditable}
+                            />
+                        </FormLabelGroup>
+                    )}
+                    <FormLabelGroup
+                        isRequired={values.updateCredentials}
+                        label="Client ID"
+                        labelIcon={
+                            <IntegrationHelpIcon
+                                helpTitle="Service account client ID"
+                                helpText={
+                                    <div>
+                                        Client identifier for a{' '}
+                                        <Button variant="link" isInline>
+                                            <a
+                                                href="https://console.redhat.com/iam/service-accounts"
+                                                target="_blank"
+                                                rel="noreferrer"
+                                            >
+                                                Red Hat service account
+                                            </a>
+                                        </Button>
+                                        . The service account must belong to a group with the{' '}
+                                        <strong>Subscription Viewer</strong> role. See{' '}
+                                        <Button variant="link" isInline>
+                                            <a
+                                                href="https://docs.redhat.com/en/documentation/openshift_cluster_manager/1-latest/html-single/managing_clusters/index#assembly-user-management-ocm"
+                                                target="_blank"
+                                                rel="noreferrer"
+                                            >
+                                                Configuring access to clusters in OpenShift Cluster
+                                                Manager
+                                            </a>
+                                        </Button>{' '}
+                                        for more information.
+                                        <br />
+                                        <br />
+                                        <em>
+                                            Service accounts are the preferred authentication method
+                                            over the deprecated API token.
+                                        </em>
+                                    </div>
+                                }
```

Comment:
Related to `a` elements for external links.

Although, hard to say if useful for links in sentences, there is `ExternalLink` element to provide the conventional icon.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Components/PatternFly/IconText/ExternalLink.tsx

Example on System Health:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/SystemHealth/CertificateHealth/CertificateCard.tsx#L133-L144

Reply:
Thanks for educating me on how PatternFly are supposed to be used. I opted against the icon since I think it renders better without in full sentences.

Reply:
As David said, the team will soon replace `Button` element with `component="a"` prop, which has only 2 occurrences (not including this contribution). I will suggest alternatives in code comments.

Reply:
For your information: new lint rule for **RHACS docs** links, although not other links as in this contribution:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/eslint-plugins/pluginGeneric.js#L127-L137